packages/@sfcc-bff/basketapi/src/api/schema/basketResolvers.ts

```diff
@@ -14,11 +14,10 @@ import {
 } from '@sfcc-core/core-graphql';
 import CommerceSdk from 'commerce-sdk';
 import { BasketT } from 'commerce-sdk/dist/checkout/shopperBaskets/shopperBaskets.types';
-import { core, Config } from '@sfcc-core/core';
+import { Config } from '@sfcc-core/core';
 import { getPrices } from '@sfcc-bff/productapi';
 
 const { ApolloError } = apollo;
-const logger = core.logger;
```

Comment:
clean up: removed this because we are not using it anywhere after removing catch statements